apps/storybook/src/SearchControls.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { text } from '@storybook/addon-knobs';
+
+import { SearchControls } from '@asap-hub/react-components';
+import { action } from '@storybook/addon-actions';
+
+export default {
+  title: 'Organisms / Search Controls',
+  component: SearchControls,
+};
+
+export const Normal = () => (
+  <SearchControls
+    placeholder={text('Placeholder', 'Search for a protein, a method…')}
+    onChangeSearch={() => action('Search')}
```

Comment:
Can we go with either `search` or `query` as the terminology, or even `searchQuery` I guess?

Reply:
This changes allot of files. Mind if i do it in my next PR?

Reply:
ok